### ISSUE TITLE
Add shutdown procedure

### ISSUE THREAD

**Issue by MartijnBraam** [dreemurrs-embedded/RescueSD#2]
No way to safely shut down the flasher yet.

**Comment by Danct12** [comment]
Maybe we can press the power button for 3 times to trigger a safe shutdown?

**Comment by Arian-D** [comment]
This is not a complete solution, but passing `conv=fdatasync` to `dd` and/or running `sync` after the image has been written could potentially ensure that no data is left in the cache.

**Comment by MartijnBraam** [comment]
Yes shutting down isn't really required, the writes aren't buffered so as long as you safely eject the phone's virtual disk after writing everything should be fine.
